.changeset/calm-pots-turn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+Remove startGlobal option
```

Comment:
If they're public API the changelog should probably mention the new events too, even if we don't want to deal with documenting them yet.

Reply:
done